proposals/NNNN-Avoid-Deadlock.md

```diff
@@ -0,0 +1,31 @@
+# Add a mechanism to avoid Deadlock
+
+* Proposal: [SDL-NNNN](NNNN-Avoid-Deadlock.md)
+* Author: [Yuki Shoda](https://github.com/Yuki-Shoda)
+* Status: Awaiting review
+* Impacted Platforms: [iOS]
+
+## Introduction
+This proposal is to avoid deadlock by adding a mechanism that stops AudioStreaming under certain conditions when the app on HS side moves to background.
+
+## Motivation
+In a combination of HU, which uses SDL Core4.5,  and sdl_ios (v6.3.1 or later), a DeadLock may occur when AudioStreaming is played while VideoStreaming is stopped.
+Due to this support (https://github.com/smartdevicelink/sdl_ios/pull/1235/) in the current sdl_ios (v6.3.1 or later),  AudioStreaming continues to play when the app on HS side moves to Background.
+For this reason (#1235), a deadlock may occur.
+To avoid the deadlock, it is desirable to add a mechanism where the protocol version is judged by the Proxy and stops AudioStreaming when the app on HS side move to background.
```

Comment:
```suggestion
To avoid the deadlock, it is desirable to add a mechanism where the protocol version is judged by the proxy and stops Audio Streaming when the app on HS side move to background.
```
Please specify the "HS" reference